### DISCUSSION TITLE
Changing meetings from live to Zoom only - then shows Temporarily Inactive

### DISCUSSION THREAD

**Discussion by wayonline63** [code4recovery/12-step-meeting-list#563] [Q&A] [answered]
### Contact Details

aawebmailwvi@gmail.com

### Website With Issue

www.aasalem.org

### What happened?

A bug happened!

[Bug]: Changing meetings from live to Zoom only - then shows Temporarily Inactive

I looked through the information and cannot find anything. I also deactivated as many plugins as possible to see if that helped. In addition, I rolled back WP from 5.8.1 to 5.8 and that didn't seem to work, either.

![image](https://user-images.githubusercontent.com/74514911/139965330-2fd2f769-5f6d-4edb-99b2-ba876a1251b1.png)

Please advise!

Thanks, Sarah

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Chrome

### Relevant log output/errors

```shell
We still have not moved the site from our old webmaster's personal server to a commercial server, although we are working on that. So I can't get to the files the usual way. Is there any other way to generate the error logs?

Thank you,
Sarah
```

**Comment by tech2serve** [comment, marked as answer]
Hi Sarah—

For a natively online meeting, the address must be approximate (i.e., no street) reflecting the city (or possibly county) and state only. By leaving a complete address but telling the app you don’t want meet in-person, you’re indicating you want the location to be treated as temporarily inactive, that is why it shows that way. This is not a bug, it is expected.

To resolve, simply change the address to be just the city and state, and you should see the temporarily inactive flag clear.

**Reply by wayonline63** [reply]
Hi!

Okay, so we tried that and still no go. The meeting still shows inactive.

I added another meeting with the zoom only and Salem, OR, but no other address details, and still inactive.

Here is an admin login for you should you prefer that - 
Username - code4recovery
email - notifications@github.com
pw - *i*)i%1V7GBMbFfNjjjQSq%U

The meeting(s) we have been trying to correct are multiple days of the week and are named "Salem 7am Zoom Group" and you can see that when we change the address to city Salem, OR and say it is not an in-person meeting we are still getting the Inactive notice on the meeting.

Please let me know what you need from me.
In Service, Sarah

**Reply by tech2serve** [reply]
The login didn't work, but that is okay as this is a public site and anyone reading it could log into your site. Not a good idea. If we need to have one of the @code4recovery/twelve-step-meeting-list devs look at it, we'll exchange credentials out of public view, or set up a TeamViewer or Zoom session.

I suspect you don't have the Zoom meeting information as a valid link in the Conference URL field. This is required for an online meeting. You can't simply put it in the notes or somewhere else. It (or the Conference Phone field) must be completed and valid. And again, you must only have the city and state for the address if you want the meeting to reflect as _only_ online. 

If I am incorrect about that, please reply to this thread with a couple of screen caps of your admin edit meeting page so we can look at that. 

Also, you are getting some errors on your site (look at the javascript/developer console) that *could* be affecting things. I suggest trying to clear those up. None appear to be directly related to TSML.

**Comment by wayonline63** [comment]
Okay, thank you!!

I just couldn't figure it out and read that we needed the address. Sorry I didn't see the part about using a general address.

We will give it a try!

Warmly,
Sarah

**Comment by wayonline63** [comment]
That corrected it, thank you.
